ceremonies/Cargo.toml

```diff
@@ -22,8 +22,7 @@ frame-support = { version = "4.0.0-dev", git = "https://github.com/paritytech/su
 pallet-timestamp = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", default-features = false, branch = "master" }
 sp-runtime = { version = "6.0.0", git = "https://github.com/paritytech/substrate.git", default-features = false, branch = "master" }
 sp-std = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", default-features = false, branch = "master" }
```

Comment:
Above section == here.

Also, please re-introduce the deleted `# benchmarking` on line 26.